### DISCUSSION TITLE
Python Integration

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/interactive#1893] [General]
Hello dotnet interactive team. My team and I absolutely love this tool! We use a lot of PowerShell and SQL notebooks, often together. The big feature we feel is missing at the moment is Python integration. Once python is added, dotnet interactive will likely replace our existing Jupyter notebooks as our one-stop notebook solution. You mention that Python integration is being investigated. I'm wondering if you can drop any clues as to when we might expect to see this or if it is actively being worked on.

**Comment by GeorgeS2019** [comment]
FYI 
- [PythonNET integration](https://github.com/dotnet/interactive/issues/2126)
- [PythonNET integration with .NET Interactive notebook ](https://github.com/pythonnet/pythonnet/issues/1851)Need to explore pythonnet

[Area-python](https://github.com/dotnet/interactive/issues?q=is%3Aissue+is%3Aopen+python+label%3AArea-Python)
